### DISCUSSION TITLE
local instance with nginx proxy manager cant request SSL certificate

### DISCUSSION THREAD

**Discussion by zqbake** [nextcloud/all-in-one#3297] [Questions] [answered]
Failed to request a SSL certificate in nginx proxy manager with the domain `nextcloud.lan`.

Does this mean I need to purchase a domain in order to set up a locally accessible instance?
Why are we being forced to use a domain?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/blob/main/local-instance.md
